platform/pulpcore/tasking/tasks.py

```diff
@@ -243,13 +243,22 @@ def apply_async(self, tags=[], group_id=None, args=None, kwargs=None, **options)
 
     def __call__(self, *args, **kwargs):
         """
-        Set the :class:`pulpcore.app.models.Task` object in the running state and log some output.
+        Set the :class:`pulpcore.app.models.Task` object in the running state, set the worker,
```

Comment:
This somehow needs to be condensed to a single line. The first line of a multi-line docstring should be a single line with a line of whitespace after to adhere to [PEP257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).